src/Umbraco.Cms.Api.Management/Services/FileSystem/FileSystemTreeServiceBase.cs

```diff
@@ -73,7 +73,7 @@ public string[] GetDirectories(string path) => FileSystem
         .OrderBy(directory => directory)
         .ToArray();
 
-    public string[] GetFiles(string path) => FileSystem
+    public virtual string[] GetFiles(string path) => FileSystem
```

Comment:
Unfortunately this [is considered a binary breaking change](https://learn.microsoft.com/en-us/dotnet/core/compatibility/library-change-rules#modifications-to-the-public-contract) - which despite my comment on the behavioural breaking change being discussed - is something we have to avoid.  

I think you could get around this though by making this amend in `FileSystemTreeServiceBase`:

```
    public string[] GetFiles(string path) => FileSystem
        .GetFiles(path)
        .Where(FilterFile)
        .OrderBy(file => file)
        .ToArray();

    protected virtual bool FilterFile(string file) => true;
```

And then in the derived `StyleSheetTreeService`:

```
    protected override bool FilterFile(string file) => file.ToLowerInvariant().EndsWith(".css");
```